### ISSUE TITLE
Explanation on the Predict Output Needed

### ISSUE THREAD

**Issue by MattYoon** [xuxingya/tf2crf#22]
Running `model.predict()` returns tuple with 4 indices.

I understand that the index 0 is the classification result, but it's not clear what the rests are.
An explanation on the rest 3 indices would be greatly appreciated.

**Comment by xuxingya** [comment]
I think there will only be one output when running model.predict().
The layer will have different output in training or in predicting.
The four outputs when training are:
viterbi_sequence: the predicted sequence tags
inputs: the input logits of the CRF layer
sequence_lengths: True sequence length for masking
self.transitions: the internal transition parameters of CRF
The one output in predicting is viterbi_sequence

**Comment by MattYoon** [comment]
Thank you for the explanation.
I double checked, and `model.predict()` does return four outputs.
I am running the newest version.

**Comment by xuxingya** [comment]
Yes, you are right. It should be 4 outputs. I forget that is defined in
base mode.
